.circleci/config.yml

```diff
@@ -13,10 +13,10 @@
 #
 #   * TERMINUS_MACHINE_TOKEN
 #
-# It also depends on the following environment variables declared in docker/drupal.env:
+# It also depends on the following environment variables declared in .env/drupal.env:
```

Comment:
I don't think we should use .env as a directory, because it prevents us from ever using a .env file.  In #111, we discussed moving this into `.lando` instead.